pkg/sentry/kernel/fd_table.go

```diff
@@ -573,7 +585,7 @@ func (f *FDTable) GetVFS2(fd int32) (*vfs.FileDescription, FDFlags) {
 // Precondition: The caller must be running on the task goroutine, or Task.mu
 // must be locked.
 func (f *FDTable) GetFDs(ctx context.Context) []int32 {
-	fds := make([]int32, 0, int(atomic.LoadInt32(&f.used)))
+	fds := make([]int32, 0, int(f.fdBitmap.GetNumOnes()))
```

Comment:
We'll need to lock before this since `f.fdBitmap` is protected by `f.mu`:
```go
f.mu.Lock()
defer f.mu.Unlock()
```

Reply:
How about applying atomic in bitmap library?

Reply:
I'm not sure how we could use an atomic here. We need the lock both to call `GetNumOnes` and to iterate over the list of FDs in `f.forEach`.